src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_application.py

```diff
@@ -89,7 +89,7 @@ def _app_service_test(self):
 
     @ResourceGroupPreparer()
     @KeyVaultPreparer(name_prefix='sfrp-cli-kv-', additional_params='--enabled-for-deployment --enabled-for-template-deployment')
-    def test_application(self, key_vault):
+    def test_application_related(self, key_vault, resource_group):
```

Comment:
It seems there is potential issue in "azdev" that even if I add `--discover` "azdev" still cannot discover the test `test_application`.
It's a workaround to fix the issue by renaming the test.